### ISSUE TITLE
SElinux support on chroot options

### ISSUE THREAD

**Issue by adrian15** [rescatux/rescatux#39]
1) See if we need SElinux support at kernel level in order to be able to mount filesystems to that it's SElinux capabilities are detected with -t auto switch . Maybe removing -t auto should do the trick. I'm not sure.

Once SElinux is in the live cd check if grub-install or update-grub respect the original SElinux permissions or not.

An example of why might be needed to do when you use chpasswd from the Rescatux live cd:
```
restorecon -v /etc/shadow
restorecon reset /etc/shadow context system_u:object_r:file_t:s0-system_u:object_r:shadow_t:s0
```

**Comment by adrian15** [comment]
(When the bug will be fixed)
Inform: Dr. Chris Brown from Linux Format magazine. His January 2014 Linux Format UK article reported this bug.

His email is:

chris.linuxformat AT gmail DOT com

**Comment by adrian15** [comment]
It would seem that Debian disables SELinux by default but that the kernel is SELinux capable by default.

https://wiki.debian.org/SELinux/Setup

Some packages are recommended but I'm not sure we will need them because I don't want selinux in Rescatux per se but when mounting ext4 partitions.

What we need to test is adding:
```
selinux=1 enforcing=0
```
at boot parametres which would enable selinux in permissive mode.

So, let's test if that it's enough for being able to edit selinux systems in enforced mode without loosing its original selinux security contexts.

**Comment by adrian15** [comment]
Fixed at 0.40 beta 1.

Commits:
eb014df319a390abc3efd0682ca14a74b33b461e
4318f18b61901d898ca24734f578a7846b14dc0d
3237938e62e37c62cd03575e5be0c349e3d923e5
7b9c95246d56fc539ad2e0cb899e83b7fe2c7518
9f74111d7c5222a739054af1900784481f6496c3
